src/CMakeLists.txt

```diff
@@ -22,23 +22,29 @@ set_target_properties           (leptonica PROPERTIES OUTPUT_NAME       leptonic
 set_target_properties           (leptonica PROPERTIES DEBUG_OUTPUT_NAME leptonica-${VERSION_PLAIN}d)
 endif()
 
-if (NOT STATIC)
+if (BUILD_SHARED_LIBS)
     target_compile_definitions  (leptonica PRIVATE -DLIBLEPT_EXPORTS)
 endif()
 
+target_include_directories(
+    leptonica
+    INTERFACE
+    "$<INSTALL_INTERFACE:include/leptonica>"
```

Comment:
I'll have a look at how hard it is to fix tesseract.
A quick search shows #include <allheaders.h> used in about ~60 files.
https://github.com/tesseract-ocr/tesseract/search?utf8=%E2%9C%93&q=allheaders.h&type=

Is it possible to extract the include dirs from the Leptonica target and then add 'leptonica' to it and then include_directories on that inside tesseract?